specification/azsadmin/resource-manager/update/Microsoft.Update.Admin/preview/2021-07-01/Updates.json

```diff
@@ -0,0 +1,461 @@
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Update.Admin/updateLocations/{updateLocation}/updates/{updateName}": {
+      "get": {
+        "x-ms-examples": {
+          "Get a specific update at an update location.": {
+            "$ref": "./examples/Updates/Get.json"
+          }
+        },
+        "tags": [
+          "Updates"
+        ],
+        "description": "Get a specific update at an update location.",
+        "operationId": "Updates_Get",
+        "parameters": [
+          {
+            "$ref": "Update.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "Update.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "UpdateLocations.json#/parameters/UpdateLocationParameter"
+          },
+          {
+            "$ref": "Updates.json#/parameters/UpdateParameter"
+          },
+          {
+            "$ref": "Update.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Update"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Update.Admin/updateLocations/{updateLocation}/updates/{updateName}/Prepare": {
+      "post": {
+        "x-ms-examples": {
+          "Prepare a specified update at an update location.": {
+            "$ref": "./examples/Updates/Prepare.json"
+          }
+        },
+        "tags": [
+          "Updates"
+        ],
+        "description": "Prepare a specified update at an update location.",
+        "operationId": "Updates_Prepare",
+        "parameters": [
+          {
+            "$ref": "Update.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "Update.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "UpdateLocations.json#/parameters/UpdateLocationParameter"
+          },
+          {
+            "$ref": "Updates.json#/parameters/UpdateParameter"
+          },
+          {
+            "$ref": "Update.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "UpdateRuns.json#/definitions/UpdateRun"
+            }
+          },
+          "202": {
+            "description": "ACCEPTED",
+            "schema": {
+              "$ref": "UpdateRuns.json#/definitions/UpdateRun"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Update.Admin/updateLocations/{updateLocation}/updates/{updateName}/HealthCheck": {
```

Comment:
Shouldn't prepare and health check be in a single API?

Reply:
They are actually different operations. Prepare triggers download + health check, but healthcheck only do health check, and you can rerun health check as many times as you want, but Prepare, most likely only one time.